### ISSUE TITLE
Add View and TouchableWithoutFeedback accessibilityIgnoresInvertColors

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-windows#8189]
https://reactnative.dev/docs/view#accessibilityignoresinvertcolors

From [RN blog](https://reactnative.dev/blog/2018/08/13/react-native-accessibility-updates#solution-to-problem-three) - "Some users with vision loss use inverted colors on their mobile phones to have greater screen contrast. Apple provided an API for iOS which allows developers to ignore certain views. This way, images and videos aren't distorted when a user has the inverted colors setting on. This API is currently unsupported by React Native."

This is an iOS-only prop -- - do we want to bring this to RNW?

**Comment by chrisglein** [comment]
Sounds like it's to opt out some Apple behavior. The question is if High Contrast mode on Windows has something similar... and if so if this property should map to the same behavior.

Let's say this doesn't work by design, but we'll reactivate this issue if we get requests otherwise.
